provisioning/e2e/package.json

```diff
@@ -10,13 +10,17 @@
     "azure-iot-provisioning-device-amqp": "0.0.1",
     "azure-iot-provisioning-device-http": "0.0.1",
     "azure-iot-provisioning-device-mqtt": "0.0.1",
-    "debug": "^3.0.1",
-    "machina": "^2.0.0",
+    "azure-iot-provisioning-service": "0.0.1",
+    "azure-iothub": "^1.1.18",
```

Comment:
>1.1.18 [](start = 22, length = 6)

we're now on 1.2.0 - also please make sure you add this to the input.js file in the release repository.